src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -96,6 +107,15 @@ function ProviderSelectionField({
               >
                 Change provider
               </button>
+              <button
+                type="button"
```

Comment:
Probably makes sense to have an aria label that says "Remove provider" or even `Remove ${formData.name}`

Reply:
![image](https://user-images.githubusercontent.com/8315447/101549947-8846c780-397c-11eb-87f4-d83b47cc7ecb.png)

Reply:
@jbalboni once you confirm the above fix I will merge it

Reply:
Looks good to me